website/docs/patterns/objects.md

```diff
@@ -33,6 +33,10 @@ class PostgreSQLConnection(DBConnection):
         self.user = user
         self.password = password
         self.database = database
+    
+    @classmethod
+    def default_init_method(self, database):
+        return self("localhost", "admin", "password", database)
```

Comment:
I think this can be removed now.

Reply:
I propose that you add a new section to this page describing `hydra.utils.instantiate()`.
It's probably should come near the end, after we introduce it for the first time.